devices.js

```diff
@@ -8875,6 +8875,30 @@ const devices = [
             });
         },
     },
+    
+    // Develco 
+    {
+        zigbeeModel: ['SMSZB-120'],
+        model: 'SMSZB-120',
+        vendor: 'Develco Products A/S',
+        description: 'Smoke detector with siren',
+        supports: 'smoke, warning, temperature',
+        fromZigbee: [fz.temperature, fz.bitron_battery_att_report, fz.develco_smoke, fz.ignore_basic_report, fz.ignore_genOta],
+        toZigbee: [tz.warning],
+        meta: {configureKey: 1},
+        configure: async (device, coordinatorEndpoint) => {
+            const endpoint = device.getEndpoint(35);
+            await bind(endpoint, coordinatorEndpoint, ['genPowerCfg', 'ssIasZone', 'genBasic']);
+            await configureReporting.batteryVoltage(endpoint);
```

Comment:
Can you check if instead of `batteryVoltage`, `await configureReporting.batteryPercentageRemaining(endpoint)` works? In fromZigbee `bitron_battery_att_report` can be replaced with `fz.battery`.

Reply:
Yeah, that also works

Reply:
Could you update the PR?